### PR TITLE
UI: Remove unused dependencies and fix travis build

### PR DESCRIPTION
**Pull request by utchoang**
### Description

This PR related https://github.com/apache/cloudstack/issues/5950

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### Screenshots (if appropriate):
image

### CODE REVIEW

**Comment by davidjumani** [comment]
@blueorangutan ui

File: ui/package.json [hunk 1]
```diff
@@ -52,11 +52,9 @@
     "npm-check-updates": "^6.0.1",
     "nprogress": "^0.2.0",
     "vue": "^3.2.31",
-    "vue-clipboard2": "^0.3.1",
     "vue-cropper": "^1.0.2",
     "vue-i18n": "^9.1.6",
     "vue-loader": "^16.2.0",
-    "vue-ls": "^3.2.2",
```
Comment (rohityadavcloud) [inline ui/package.json:59]: aren't we using localstorage now?

File: ui/package.json [hunk 2]
```diff
@@ -86,10 +84,8 @@
     "eslint-plugin-vue": "^7.0.0",
     "less": "^3.0.4",
     "less-loader": "^5.0.0",
-    "node-sass": "^4.0.0",
```
Comment (rohityadavcloud) [inline ui/package.json:89]: Don't we need node-sass for building css etc?

File: ui/package.json [hunk 3]
```diff
@@ -86,10 +84,8 @@
...
     "sass": "^1.49.9",
     "sass-loader": "^8.0.2",
-    "uglifyjs-webpack-plugin": "^2.2.0",
```
Comment (rohityadavcloud) [inline ui/package.json:92]: Don't we want to minimise/compress the UI build?

File: ui/package.json (hunk 2 above)
Comment (utchoang) [reply ui/package.json:89]: I found that removing it has no effect on the build process. I'm not sure about this, can keep it if we want.

File: ui/package.json (hunk 3 above)
Comment (utchoang) [reply ui/package.json:92]: My apologies, I tried to delete it to check the audit fix but made a mistake. Will add it back in.

**Comment by davidjumani** [comment]
@blueorangutan ui

File: ui/package.json (hunk 2 above)
Comment (rohityadavcloud) [reply ui/package.json:89]: @utchoang I suppose it's worth checking where/how it is used, esp for any `less` and related stylesheet processing. (it's possible it doesn't happen or is required for build, but maybe for development)

**Comment by utchoang** [comment]
@blueorangutan ui

**Comment by rohityadavcloud** [comment]
Thanks @utchoang can you re-run the npm install/build etc to regenerate and ensure that the package-lock.json is correct and updated as well?

**Review by rohityadavcloud — APPROVED** [review]
LGTM didn't test it.

**Comment by nvazquez** [comment]
Moving this to the next milestone to prevent any regressions near the 4.17 freeze date

**Review by rohityadavcloud — APPROVED** [review]

**Review by rohityadavcloud — APPROVED** [review]

**Review by rohityadavcloud — APPROVED** [review]

**Comment by rohityadavcloud** [comment]
UI build fix, seems good to me. Verified the UI build (qa server) and also packages.

**Pull request merged**